README.md

```diff
@@ -20,22 +20,34 @@ provided by [nim-markdown]. For a quick reference of supported syntax see the [c
 
 ### Status
 
-nimibook currently provides minimal functionality to create a book, notably missing is
-a command line interface to automate common task (e.g. initialization of a new nimibook).
+nimibook currently provides minimal functionality to create a book and support a local CLI mode.
 
 ## Installation
 
-To be able to use nimibook, currently the only way is to recreate the same content of the repository
-and adjust it to your content. You can do that using the template feature from github
-or with a checkout of the repo and copying and pasting.
-
-Discussion of further options is ongoing in [issue #12](https://github.com/pietroppeter/nimibook/issues/12).
+To install Nimibook simply use : ``nimble install nimibook``
```

Comment:
ah wait, we need to tell the user how to write the initial `nbook.nim` that will allow him to init everything!